src/Component/spec/Filter/NumericRangeFilterSpec.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Component\Grid\Filter;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Grid\Data\DataSourceInterface;
+use Sylius\Component\Grid\Data\ExpressionBuilderInterface;
+use Sylius\Component\Grid\Filtering\FilterInterface;
+
+final class NumericRangeFilterSpec extends ObjectBehavior
+{
+    function it_implements_a_filter_interface(): void
+    {
+        $this->shouldImplement(FilterInterface::class);
+    }
+
+    function it_does_nothing_when_there_is_no_data(DataSourceInterface $dataSource): void
+    {
+        $this->apply(
+            $dataSource,
+            'number',
+            [],
+            [],
+        );
+    }
+
+    function it_filters_number_from(
+        DataSourceInterface $dataSource,
+        ExpressionBuilderInterface $expressionBuilder,
+    ): void {
+        $expressionBuilder->greaterThanOrEqual('number', '3')->willReturn('EXPR');
```

Comment:
Does this work? The second argument should be an integer. 🤔

Reply:
example -> `vendor/sylius/grid-bundle/src/Bundle/spec/Doctrine/PHPCRODM/ExpressionBuilderSpec.php:78`

Reply:
Yes, works (but I changed)